src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -498,6 +500,9 @@ public CaseDTO modifyCase(ModifyCaseRequestDTO modifyRequestDTO) throws CTPExcep
     UUID caseId = originalCaseId;
 
     CaseContainerDTO caseDetails = getCaseFromRmOrCache(originalCaseId, true);
+    if (!CENSUS.equalsIgnoreCase(caseDetails.getSurveyType())) {
```

Comment:
potential for this to be factored into a resusable method  **validateSurveyType()**  and use appConfig.getSurveyName().

Reply:
Done: method added